src/ep.config.json

```diff
@@ -32,7 +32,7 @@
   "enableOfflineMode": false,
   "ImageContainerSrcs": {
     "types": ["webp", "jp2", "jpg", "jxr"],
-    "sizes": ["768", "1092", "2800"]
+    "sizes": [["768", "620x620"], ["1092", "490x490"], ["2800", "450x450"]]
```

Comment:
We should not change the settings in ep.config.json because not every image will be that size at a given screen breakpoint.  Ex.  at 768w we could have an image that is 768x200 or one at 200x200.

Reply:
Instead we can just have the script take an argument with what is in the ep.config.json.  Or it its too long maybe create a seperate json file just for processing images.  Something that specifies a list of directories, breakpoints, and image sizes at each breakpoint.